rollup.config.js

```diff
@@ -32,11 +33,15 @@ const config = {
     commonjs({
       include: ['node_modules/**'],
     }),
-    babel({
-      babelrc: true,
-      exclude: ['node_modules/**'],
-      runtimeHelpers: true,
-    }),
+    babel(
+      Object.assign(
+        {
```

Comment:
I thought you could spread here. Not sure what the nvmrc says in the repo these days 🎃

Reply:
I believe that this file is not getting transpiled by babel and as we support node from version v6 we should not use any of new syntax here. Also, the spread operator based on ES6 works only with arrays.

Reply:
Yup, it doesn't. 

I tried ES7 Object spread first but it failed on Node6 :(